lib/Optimizer/Transforms/CMakeLists.txt

```diff
@@ -34,9 +36,11 @@ add_cudaq_library(OptTransforms
   GlobalizeArrayValues.cpp
   LambdaLifting.cpp
   LiftArrayAlloc.cpp
+  LiftArrayAllocPatterns.cpp
```

Comment:
I'm not sure why these are being split into two files in this PR?

Reply:
I replaced this by using a LiftArrayAllocPatterns.inc, used in LoopUnroll.cpp as well